logging/google/cloud/logging/client.py

```diff
@@ -40,6 +41,12 @@
 from google.cloud.logging.connection import _LoggingAPI as JSONLoggingAPI
 from google.cloud.logging.connection import _MetricsAPI as JSONMetricsAPI
 from google.cloud.logging.connection import _SinksAPI as JSONSinksAPI
+from google.cloud.logging.handlers import CloudLoggingHandler
+from google.cloud.logging.handlers import GAEHandler
+from google.cloud.logging.handlers import GKEHandler
+from google.cloud.logging.handlers import setup_logging
+from google.cloud.logging.handlers.handlers import EXCLUDE_LOGGER_DEFAULTS
```

Comment:
Why not put `EXCLUDE_LOGGER_DEFAULTS` in `google/cloud/logging/handlers/__init__.py`?